python/sunbeam/schedule.py

```diff
@@ -16,6 +16,22 @@ def wells(self):
     def group(self, name):
         return Group(self._group(name), self)
 
+    def group_tree(self, timestep):
+        return GroupTree(self._group_tree(timestep))
+
+    def gtree(self, timestep):
+        '''Fuction to return python struct of group topologi for given timestep'''
+        gt_obj = self._group_tree(timestep)
+        gt={}
```

Comment:
Exactly one space on each side of `=`.  (Run code through `pylint` to get help.)